pod-operation/src/state.rs

```diff
@@ -0,0 +1,26 @@
+use std::sync::{Arc, Mutex};
+use lazy_static::lazy_static;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum State {
+    Start,
+    Stop,
+    ForceStop,
+    Load,
+    Init,
+}
+
+#[derive(Debug)]
+pub struct CurrentState {
+    pub value: Option<State>,
+}
+
+impl CurrentState {
+    pub fn new(value: Option<State>) -> Self {
+        CurrentState { value }
+    }
+}
```

Comment:
Question: is there a particular reason we have this struct and also why the value is optional? We should be able to have the state as just a field of the FSM struct, and it wouldn't make sense for the state to be `None` (see #21).